common/config.h.in

```diff
@@ -0,0 +1 @@
+#cmakedefine NLOHMANN_JSON @NLOHMANN_JSON@
```

Comment:
I prefer
```
#define MENDER_COMMON_USE_NLOHMANN_JSON 1
```
over just
```
#define MENDER_COMMON_USE_NLOHMANN_JSON
```
because the former work with both `#ifdef` and `#if`.